docs/modules/Either.ts.md

```diff
@@ -346,12 +404,24 @@ Added in v2.6.0
 
 ## flatten
 
+The `flatten` function is the conventional monad join operator. It is used to remove one level of monadic structure, projecting its bound argument into the outer level.
+
 **Signature**
 
 ```ts
 export declare const flatten: <E, A>(mma: Either<E, Either<E, A>>) => Either<E, A>
 ```
 
+**Example**
+
+```ts
+import * as E from 'fp-ts/Either'
+
+assert.deepStrictEqual(E.flatten(E.right(E.right('a'))), E.right('a'))
```

Comment:
as it is removing one level only do you think this is a good example?

assert.deepStrictEqual(E.flatten(E.right(E.right(E.right('a')))), E.right(E.right('a')))